### PR TITLE
Add team onboarding section

### PR DESCRIPTION
**Pull request by danielm0hr**
This is a cleaned up version of the internal
onboarding document
(https://docs.google.com/document/d/1bJSrlyc-e7bcOxV4sjx3FesMNVgdwNxUzMvIYywbt-0)
with the intention to replace it. It still has many
links to internal documents but is thought to serve
as the top level starting point for people onboarding
to Red Hat OpenShift Monitoring teams.

Over time more and more of the internal links can be
moved to this repository as well.

### CODE REVIEW

File: content/Teams/onboarding/README.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+
+Welcome to the Monitoring Team! We created this document to help guide you through your onboarding.
+
+Please fork this repo and propose changes to the content as a pull request if something is not accurate, outdated or unclear. Use your fork to track the progress of your onboarding tasks, i.e. by keeping a copy in your fork with comments about your completion status. Please read the doc thoroughly and check off each item as you complete it. We consistently want to improve our onboarding experience and your feedback helps future team members.
+
+This documents contains some links to Red Hat internal documents and you won't be able to access them without a Red Hat associate login. We are still trying to keep as much information as possible in the open.
+
+## Team Background
+
+Team Monitoring mainly focuses on Prometheus, Thanos, Observatorium, and their integration into Kubernetes and OpenShift. We are also responsible for the hosted multi-tenant Observability service which powers such services as OpenShift Telemetry and OSD metrics.
```
Comment (raptorsun) [inline content/Teams/onboarding/README.md:35]: better expand OSD to its full name

File: content/Teams/onboarding/README.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+CoreOS was acquired by Red Hat in early 2018. The CoreOS monitoring team became the Red Hat Monitoring Team, which has evolved into the “Observability group”. The teams were divided into two teams:
+
+1. The In-Cluster Observability Team
+2. The Observability Platform team (aka RHOBS or Observatorium team)
+
+You may encounter references to these two teams. In early 2021 we decided to combine the efforts of these teams more closely in order to avoid working in silos and ensure we have a well functioning end-to-end product experience across both projects. We are, however, still split into separate scrum teams for efficiency. We are now collectively known as the “Monitoring Team” and each team works together across the various domains.
+
+We work on all important aspects of the upstream eco-system and a seamless monitoring experience for Kubernetes using Prometheus. Part of that is integrating our open source efforts into the OpenShift Container Platform (OCP), the commercial Red hat Kubernetes distribution.
+
+## People relevant to Prometheus who you should know about:
```
Comment (bwplotka) [inline content/Teams/onboarding/README.md:48]:
```suggestion
## Some people relevant to Prometheus who you should know about:
```

File: content/Teams/onboarding/README.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+
+After your starter project is done, we’ll discuss how it went and what your future projects will be. By then you'll hopefully have a good overview which areas you are interested in and what their priority is. Discuss with your team lead or manager what your next project will be.
+
+## Glossary
+
+Below is a glossary of terms and acronyms which may be unfamiliar to you.
+
+If what you’re looking for is missing, consider checking the [Abbreviations, Acronyms, and Initialisms Dictionary](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon) page on the Source.
+
+<table>
```
Comment (bwplotka) [inline content/Teams/onboarding/README.md:253]: Should we move it to Markdown?

**Review by bwplotka — APPROVED** [review]
Amazing! Happy to merge this, tiny comments

**Review by jan--f — COMMENTED** [review]
I'd be +1 for merging this as is. Improvements will follow.

File: content/Teams/onboarding/README.md [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+
+### Administrative things
+
+1. Review all the onboarding instructions/documentation sent to your email by the People Team (HR).
+2. Submit an[Employee Badge Request Form](https://redhat.service-now.com/rh_ess/cat_item.do?&sysparm_document_key=sc_cat_item,ac8bb875a82282004c7185ae62325874)
+3. Register for new hire orientation NHO (you will get an email about this from HR)
+4. Speak with your manager about any additional equipment you need (e.g. keyboard, mouse, etc)
+5. Login to Rover (our company directory) and update your profile [https://rover.redhat.com/people/profile/](https://rover.redhat.com/people/profile/). Be sure to do the following:
+   * Add a short bio
+   * Bookmark Rover for future reference
```
Comment (simonpasquier) [inline content/Teams/onboarding/README.md:142]: It's also recommended/required to link your GitHub profile

File: content/Teams/onboarding/README.md [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+6. Add yourself in [https://spaces.redhat.com/display/OBS/Monitoring+Team](https://spaces.redhat.com/display/OBS/Monitoring+Team)
+
+### Meetings
+
+Ask your manager to be added to the [Observability Program](https://calendar.google.com/calendar/u/0?cid=cmVkaGF0LmNvbV91N3YwbGt2cnRuM2wwbWJmMnF2M2VkMm12MEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t) calendar. Ensure you attend the following recurring meetings:
+
+* Team syncs
+* Sprint retro/planning
+* Sprint reviews
+* Weekly architecture call
```
Comment (simonpasquier) [inline content/Teams/onboarding/README.md:152]: we don't do this anymore but we have office hours for Observatorium, prometheus operator and observability operator (at least)

File: content/Teams/onboarding/README.md [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+* [(Internal) How to Get Reviewers to Block your Changes](https://drive.google.com/file/d/1KOWv5A2qAoO1CfbfhCIJW1wcOnGpfTST/view), March 2022 by Assaf
+
+### [optional] Additional information & Exploration
+
+* [https://www.linkedin.com/learning/](https://www.linkedin.com/learning/)
+  * Red Hat has a corporate subscription to LinkedIn Learning that has great introductory courses to many topics relevant to our team
+* [Prometheus Monitoring](https://www.youtube.com/channel/UC4pLFely0-Odea4B2NL1nWA/videos) channel on Youtube
+* PromLabs [PromQL cheat sheet](https://promlabs.com/promql-cheat-sheet/)
+* [Prometheus-example-app](https://github.com/brancz/prometheus-example-appPrometheus-example-apphttps://github.com/brancz/prometheus-example-app)
+* [Kubernetes-sample-controller](https://github.com/kubernetes/sample-controller)
```
Comment (simonpasquier) [inline content/Teams/onboarding/README.md:193]: https://book.kubebuilder.io/ is probably a better option

**Review by simonpasquier — APPROVED** [review]
Added a few comments but I think it can be merged as-is and follow-up PRs can clean it up :)

**Pull request merged**